pkg/hs/urbit-king/lib/Urbit/Vere/Ames/UDP.hs

```diff
@@ -5,7 +5,8 @@
 
     - If can't open the port, wait and try again repeatedly.
     - If there is an error reading or writting from the open socket,
```

Comment:
s/writting/writing

Reply:
@pilfer-pandex this typo is still present.